slsim/Util/coolest_slsim_interface.py

```diff
@@ -0,0 +1,111 @@
+from lenstronomy.Util.coolest_interface import (
+    update_coolest_from_lenstronomy,
+    create_lenstronomy_from_coolest,
+)
+from slsim.Util.param_util import magnitude_to_amplitude, amplitude_to_magnitude
+
+
+def update_coolest_from_lenstronomy_slsim(
+    lens_class, path, file_name, band=None, mag_zero_point=27, ending="_update"
+):
+    """This function updates given coolest format .json file using lenstronomy kwargs of
```

Comment:
why do you need a lenstronomy in the function name and description? At the end you have a slsim_lens object and what you get is a coolest file. Under the hook as it is coded right now, it uses lenstronomy, but that is not important for the user using this functionality

Reply:
Changed!